pytest/tests/loadtest/locust/common/base.py

```diff
@@ -0,0 +1,327 @@
+import base64
+import json
+import base58
+import ctypes
+import locust
+import logging
+import multiprocessing
+import pathlib
+import random
+import requests
+import sys
+import time
+
+sys.path.append(str(pathlib.Path(__file__).resolve().parents[4] / 'lib'))
+
+import cluster
+import transaction
+import key
+import mocknet_helpers
+import utils
+
+from locust import HttpUser, events
+from configured_logger import new_logger
+
+DEFAULT_TRANSACTION_TTL_SECONDS = 20
+logger = new_logger(level=logging.WARN)
+
+
+class Account:
+
+    def __init__(self, key):
+        self.key = key
+        self.nonce = multiprocessing.Value(ctypes.c_ulong, 0)
```

Comment:
Am I right that nonce 0 will never be used by the load test because we will always increment it before returning in `use_nonce`?

Just trying to clarify the exact meaning of `self.nonce` here as to whether it means:
- Last used nonce
- First free nonce

I think right now due to the 0 case, it's a mix of both, but we can easily switch it to be first free nonce by renaming to "next_nonce" and returning the value that was stored before the increment.

Reply:
You can't use the 0 case at all, I just checked and got this:
```json
"InvalidTxError": {
    "InvalidNonce": {
        "ak_nonce": 0,
        "tx_nonce": 0
    }
}
```

But point taken, I will rename it to `current_nonce` to make this clear.